webapp/package-lock.json

```diff
@@ -541,6 +541,15 @@
         "node": ">=12"
       }
     },
+    "channels/node_modules/marked": {
```

Comment:
I wish I knew why NPM moves these out of `node_modules` and into `channels/node_modules` sometimes...